code/style.css

```diff
@@ -1,4 +1,297 @@
+  margin: 0px 0px 0px 5px;
+  color: grey;
+}
+
+/* Flex box adjustments for the first box and the four last boxes*/
+.box { 
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+/* Style the small news content - with GRID */
+.grid-container { /* Parent to grid-child - the smaller news */
+ margin: 10px;
+ display: grid;
+ grid-template-columns: repeat(4, 1fr);
+ grid-gap: 10px;
+}
+
+.grid-child { /* Child */
+  border: 1px solid #172A3A;
+  padding: 5px;
+}
+
+.one, .two, .three, .four, .eight {
+  grid-column: span 1;
+}
+
+.five, .six {
+  grid-column: span 2;
+}
+
+.seven {
+  grid-column: span 3;
+}
+
+.grid-child a { /*To fix the links*/
+  color: #172A3A;
+  text-decoration: none;
+}
+
+h3 { /* Heading in the small boxes*/
+  margin: 5px;
+}
+
+/* Styling for the last box with some animation */
+.eight { /* Make the box shift color */
+  animation: pulse 5s infinite;
+}
+
+@keyframes pulse {
+  0% {
+    background-color: #f5f3f0;
+  }
+  100% {
+    background-color: #968d80;
+  }
+}
+
+.eight p {
+  font-size: 20px;
+}
+
+.eight h6 { /* Make the text for 'date posted' to shift color */
+  animation: pulse2 5s infinite;
+}
+
+@keyframes pulse2 {
+  0% {
+    color: #172A3A;
+  }
+  100% {
+    color: #080f14;
+  }
+}
+
+/* To make it change color while hovering the boxes */
+div.grid-child:hover,  section.big-news:hover  {
+  background-color: #e6e2dc;
+}
+/* Adding shadows to the boxes*/
+.big-news, .grid-child { 
+  box-shadow: 2px 2px 2px 2px #d6d0c7;
+}
+
+/* Fix the size of the pictures in the small boxes */
+.img-small, .img-smaller { 
+  margin: 5px;
+  width: 305px;
+}
+
+.img-smallest {
+  margin: 5px;
+  width: 200px;
+}
+
+/* Styling of the footer */
+footer {
+  padding: 40px 0px 10px 0px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+ /******* Media queries settings *******/
+ @media screen and (max-width: 1024px) { /*Tablet devices */
+  .topnav a { /* Needed some smaller padding on smaller screen */
+    padding: 6px 4px;
+  }
+
+  /* To fix the 'special' boxes to columns*/
+  .big-news, .spc {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .spc {
+    align-items: center;
+  }
+
+  .spc img {
+    width: 100%;
+  }
+
+  /* Change of font-size in this smaller format */
+
+  h4 {
+    font-size: 14px;
+  }
+
+  p {
+    font-size: 12px;
+  }
+
+  .eight p {
+    font-size: 16px;
+  }
+
+/* Change of how the boxes span in tablet format */
+  .one, .two, .three, .four {
+    grid-column: span 2;
+  }
+  
+  .five, .six, .seven, .eight {
+    grid-column: span 4;
+  }
+
+  .img-small  { /* For better fit in tablet format */
+    width: 97%;
+  }
+
+  .img-smaller,  { /*To make it fit better in tablet format*/
+    width: 50%;
+  }
+}
+
+@media screen and (max-width: 667px) { /*Mobile devices */
+  html { /* Smaller margins needed to focus on reading the text instead*/
+    margin: 10px 10px;
```

Comment:
Since both vertical and horizontal directions has same 10px value, we can just use shorthand `margin: 10px`